### ISSUE TITLE
WHEN I OPEN BLOCK LAUNCHER THE APP CRACHES,THE VERSION OF MINECRAFT POCKET EDITION IS , 15.1 PLEASE FIX

### ISSUE THREAD

**Issue by themastercrcrafter** [zhuowei/MCPELauncher#1210]

**Comment by zedman2000** [comment]
@themastercrcrafter Next time, please search before creating a new issue. It has been stated over and over that this has not been updated for 0.15 yet. Please close this, the author is aware and if you follow him on Twitter you would see that he has been working on this for a few weeks and there has been big changes in 0.15, which is causing a longer than normal delay in updating 0.15.

**Comment by ISAAC300** [comment]
i have this problem too in android 4.3 :(

**Comment by themastercrcrafter** [comment]
Dises gotta fix SOON

**Comment by zedman2000** [comment]
@themastercrcrafter yes, I agree, but would you rather the author read your messages along with the other 100 or so, or actually work the updating so you have the fix soon?  Please close this.

**Comment by themastercrcrafter** [comment]
How i well fix it

**Comment by zedman2000** [comment]
@themastercrcrafter I am sorry, I do not understand what you mean.  If you mean, how do you close this, just click the close button.  If you mean, fix BlockLaucnher, wait for it to be updated, or revert your Minecraft version to 0.14.3, which matches with the PlayStore release of BlockLaucnher 1.12.8.

**Comment by themastercrcrafter** [comment]
I,m about to leavthe page and save it tell me some thing

**Comment by zedman2000** [comment]
I'm sorry, tell you what?  You did not ask a question that I could understand.  I asked you to clarify what you want.  I also took a guess at what you wanted and posted multiple answers.

**Comment by themastercrcrafter** [comment]
How wanna fix block launcher how?

**Comment by zedman2000** [comment]
I answered that.  Did you read my reply?

**Comment by themastercrcrafter** [comment]
Ok

**Comment by zedman2000** [comment]
Please close this issue so that we do not waste the author's time.  Let's let him keep working on the update.

**Comment by themastercrcrafter** [comment]
I don,t wanna revert minecraft to 14.3

**Comment by themastercrcrafter** [comment]
I,m going to the page

**Comment by zedman2000** [comment]
Then you have to wait.  There is NO version of BL that officially supports 0.15 at this time.  So you have to wait.  What page are you going to?  I'm confused.

**Comment by deathgram** [comment]
For me too finally get a mod working but every time i try to open block launcher it says block launcher has stopped working

**Comment by Aternix** [comment]
What a long conversion *_*
